DOCS-1647 - GA Dashboard Sections (#6735)
SumoLogic/sumologic-documentation: blog-service/2026-06-01-dashboards.md

---
title: Group Dashboard Panels into Sections (Dashboards)
image: https://assets-www.sumologic.com/company-logos/_800x418_crop_center-center_82_none/SumoLogic_Preview_600x600.jpg?mtime=1617040082
keywords:
  - dashboards
  - sections
  - panels
hide_table_of_contents: true
---

> We're excited to introduce dashboard sections, a new way to organize complex dashboards by grouping related panels into collapsible logical sections. You can name each section, set its default state to expanded or collapsed, and customize font size to keep your dashboards structured, focused, and easier to navigate. [Learn more](/docs/dashboards/panels/section).

Reviewer: ```suggestion
We're excited to introduce the ability to organize complex dashboards by grouping related panels into collapsible sections. You can name each section, set its default state to expanded or collapsed, and customize font size to keep your dashboards structured, focused, and easier to navigate. [Learn more](/docs/dashboards/panels/section).
```
